frontend/components/addcontent/AddContentPage.js

```diff
@@ -7,14 +7,12 @@ import Button from "@mui/material/Button";
 import ContentAdd from "./ContentAdd";
 import Footer from "../detail/Footer";
 
-export default function AddContentInfoPage({ routerQuery }) {
+export default function AddContentPage({ routerQuery }) {
```

Comment:
Actually, it may be good replacing `Content` with `Resource` in function but not crucial.